assets/stylesheets/components/_headings.scss

```diff
@@ -46,10 +46,20 @@ h6 {
   }
 }
 
-h1, h2, h3, h4, h5, h6 {
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
   .dark-theme & {
     $headings-color: $cr-gray-dark;
 
     color: $headings-color;
   }
+
+  a,
+  a & {
+    color: $cr-gray-dark;
```

Comment:
Also, there's a class of `.card-title` that's declaring the text color on the card headers. Wondering if this needs to exist anymore? The `color: inherit` stuff might conflict with this rule.